params/config.go

```diff
@@ -336,11 +336,12 @@ type ChainConfig struct {
 	EWASMBlock  *big.Int `json:"ewasmBlock,omitempty"`  // EWASM switch block (nil = no fork, 0 = already activated)
 
 	// Various consensus engines
-	Ethash   *EthashConfig   `json:"ethash,omitempty"`
-	Clique   *CliqueConfig   `json:"clique,omitempty"`
-	Istanbul *IstanbulConfig `json:"istanbul,omitempty"` // Quorum
-	IBFT     *IBFTConfig     `json:"ibft,omitempty"`     // Quorum
-	QBFT     *QBFTConfig     `json:"qbft,omitempty"`     // Quorum
+	Ethash      *EthashConfig   `json:"ethash,omitempty"`
+	Clique      *CliqueConfig   `json:"clique,omitempty"`
+	Istanbul    *IstanbulConfig `json:"istanbul,omitempty"`    // Quorum
+	IBFT        *IBFTConfig     `json:"ibft,omitempty"`        // Quorum
+	QBFT        *QBFTConfig     `json:"qbft,omitempty"`        // Quorum
+	Transitions []Transition    `json:"transitions,omitempty"` // Quorum
```

Comment:
as it is not a consensus engine, should we move it in quorum specific settings?

Reply:
updated